setup.py

```diff
@@ -43,10 +43,9 @@
     license="Apache 2",
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
-    install_requires=["pytket ~= 1.23", "pysimplex ~= 1.4"],
+    install_requires=["pytket ~= 1.24", "pysimplex ~= 1.4"],
```

Comment:
Should now require `pysimplex ~= 1.5`. (Also worth adding this to changelog.)

Reply:
done & added to changelog